.pluginsignore

```diff
@@ -1 +1,6 @@
 # Specify plugin names to ignore
+KAFKABATCHSOURCE
```

Comment:
Should we move this file under scripts/plugin..

Reply:
I can move it. My initial thought was to hold this file at the root level directory along with site configuration files. Let me know if I need to move it

Reply:
yeah I think let's move it, since it only applies to a singe script.

Reply:
done